frontend/src/components/jobs/result/utils/sections.jsx

```diff
@@ -111,6 +112,11 @@ export function JobActionsBar({ job }) {
 }
 
 export function JobInfoCard({ job }) {
+  
+  // eslint-disable-next-line camelcase, no-unused-vars
+const process_time2 = new Date(job.process_time * 1000).toISOString().substring(14, 19)
+// eslint-disable-next-line no-console
+console.log(job.process_time)
```

Comment:
can you remove debug logs after testing?

Reply:
done.